packages/components/src/button/stories/index.js

```diff
@@ -85,3 +71,35 @@ export const disabledLink = () => {
 		</Button>
 	);
 };
+
+export const buttons = () => {
+	return (
+		<div style={ { padding: '20px' } }>
+			<h2>Small Buttons</h2>
+			<div className="story-buttons-container">
+				<Button isSmall>Button</Button>
+				<Button isPrimary isSmall>Primary Button</Button>
+				<Button isDefault isSmall>Secondary Button</Button>
+				<Button isTertiary isSmall>Tertiary Button</Button>
+				<IconButton isSmall icon="ellipsis" />
```

Comment:
Not that I care that much, but something to consider.
Shouldn't be `IconButton` documented under its own component?

For reference, this is how sizes are presented for the `Icon` component.

https://wordpress.github.io/gutenberg/?path=/story/components-icon--sizes

@ItsJonQ, it's something we should standardize and wrap with some helper component so we don't have to repeat those wrappers and styles over and over again. Ideally, we could code it along those lines:

```js
<Variations label="Small Buttons">
	<Button isSmall>Button</Button>
	<Button isPrimary isSmall>Primary Button</Button>
	<Button isDefault isSmall>Secondary Button</Button>
	// ...
</Variations>
```

Reply:
to be honest, I think we can remove the IconButton component entirely in favor of an `icon` prop in the Button component (that's separate from this PR though).

the main reason I used a single story is to be able to visualize all variations in the same screen.

Reply:
> to be honest, I think we can remove the IconButton component entirely in favor of an `icon` prop in the Button component (that's separate from this PR though).

There is an issue for that for sure. I like the idea :)
You should merge this PR as soon as it's confirmed by someone from the design team :)

Reply:
> @ItsJonQ, it's something we should standardize and wrap with some helper component so we don't have to repeat those wrappers and styles over and over again.

@gziolo I think that's a great idea :). I started doing that for the [G2 Prototypes](https://github.com/WordPress/gutenberg/issues/18667) I've been making

https://tz3ir.csb.app/#/buttons

Reply:
You are always ahead of time @ItsJonQ, this is exactly what I had in mind :)